seqr/views/apis/anvil_workspace_api_tests.py

```diff
@@ -540,55 +694,60 @@ def test_create_project_from_workspace(self, mock_tempfile, mock_mv_file, mock_s
         self.assertDictEqual(json.loads(responses.calls[1].request.body), {'records': [{'fields': {
             'Requester Name': 'Test Manager User',
             'Requester Email': 'test_user_manager@test.com',
-            'AnVIL Project URL': f'http://testserver/project/{project2.guid}/project_page',
+            'AnVIL Project URL': f'http://testserver/project/{project.guid}/project_page',
             'Initial Request Date': '2021-03-01',
-            'Number of Samples': 3,
+            'Number of Samples': len(samples),
             'Status': 'Loading Requested',
         }}]})
 
-        # Test logged in locally
-        remove_token(self.manager_user)  # The user will look like having logged in locally after the access token is removed
-        response = self.client.post(url)
-        self.assertEqual(response.status_code, 302)
-        self.assertEqual(response.url, '/login/google-oauth2?next=/api/create_project_from_workspace/my-seqr-billing/anvil-no-project-workspace2/submit')
-
     @mock.patch('seqr.views.apis.anvil_workspace_api.ANVIL_LOADING_DELAY_EMAIL', 'We are unable to load your data at this time.')
     @mock.patch('seqr.views.apis.anvil_workspace_api.ANVIL_LOADING_EMAIL_DATE', '2021-06-01')
-    @mock.patch('seqr.views.apis.anvil_workspace_api.mv_file_to_gs', lambda *args, **kwargs: True)
-    @mock.patch('seqr.views.apis.anvil_workspace_api.load_uploaded_file', lambda *args, **kwargs: LOAD_SAMPLE_DATA)
-    @mock.patch('seqr.views.apis.anvil_workspace_api.logger')
-    @mock.patch('seqr.views.apis.anvil_workspace_api.datetime')
-    @mock.patch('seqr.views.apis.anvil_workspace_api.send_html_email')
     @responses.activate
-    def test_create_project_from_workspace_loading_delay_email(
-            self, mock_send_email, mock_datetime, mock_api_logger, mock_utils_logger):
+    def test_create_project_from_workspace_loading_delay_email(self):
+        responses.reset()  # Bypass dag operations
```

Comment:
I think it would be a more readable test if instead of erasing all the dag responses you explicitly added an error response from the first endpoint

Reply:
I'm going to add another task query response to allow all normal dag operations.